ViAn/Project/bookmark.cpp

```diff
@@ -68,18 +68,24 @@ std::string Bookmark::get_description() {
  * @param text
  */
 void Bookmark::set_description(const std::string& text) {
+    qDebug() << "NEW BOOKMARK DESCRIPTION " << QString::fromStdString(text);
```

Comment:
remove debug message

Reply:
The bug you mentioned, whilst important to fix, should be solved in another branch. Will add an issue.